style.css

```diff
@@ -321,3 +321,158 @@ form textarea {
 form button {
   width: 40%;
 }
+
+.desktop-menu {
+  display: none;
+}
+
+/* ---- Desktop Version ---- */
+
+@media screen and (min-width: 768px) {
+  .menu-icon {
```

Comment:
- Great Job implementing the desktop version of your portfolio. Everything looks great on screen size 1024px and 1440px 👍🏽 .
- I'd like to kindly suggest that your work on the project cards sections on screen size 768px and that represents the view of your portfolio on tablets and iPad. Can we fix this? 
- Still on screen size 768px, there is some extra space on the right side of the portfolio. 

Here is what I mean 👇🏽 

Here is what I can see compared to what's on Figma 👇🏽 
| Your Project cards  |
| :---:   | 
|<img width="606" alt="Screenshot 2023-03-30 at 11 15 52" src="https://user-images.githubusercontent.com/99246359/228902053-00acbde2-4ce2-4118-9041-0dee066175a8.png">  |

|Extra space on the right |
| :---:   | 
|<img width="601" alt="Screenshot 2023-03-30 at 11 16 10" src="https://user-images.githubusercontent.com/99246359/228902296-b8586ce4-ee52-477e-8fce-7cca1ee1e46f.png">  |

⚠️ Kindly note that the extra space issue should be fixed on the headline section of your portfolio as well. 🤗